includes/admin/helper/class-wc-helper.php

```diff
@@ -286,7 +286,7 @@ public static function render_helper_output() {
 				$data['_actions'][] = $action;
 			} else {
 				$action = array(
-					'message' => sprintf( __( 'To receive updates and support for this extension, you need to <strong>purchase</strong> a new subscription or <a href="%1$s" target="_blank">be added as a collaborator</a>.', 'woocommerce' ), 'https://docs.woocommerce.com/document/adding-collaborators/' ),
+					'message' => sprintf( __( 'To receive updates and support for this extension, you need to <strong>purchase</strong> a new subscription or consolidate your extensions to one connected account by <strong>sharing or transferring</strong> this extension to this connected account.', 'woocommerce' ), 'https://docs.woocommerce.com/document/adding-collaborators/' ),
```

Comment:
Did you mean to link to this - https://docs.woocommerce.com/document/activating-upgrading-transferring-licenses/ @belcherj